type-check/juxt.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as Test from "../src/types/Test";
+import { apply, pipe, juxt } from "../src";
+
+const { checks, check } = Test;
+
+declare function f1(a: number, b: string): string;
+declare function f2(a: number, b: string): boolean;
+declare function f3(a: number, b: boolean): number;
+declare function f4(a: number, b: string, c: boolean): number;
```

Comment:
```suggestion
// declare function f4(a: number, b: string, c: boolean): number;
```

how about comment `//` ?

Reply:
That would be better.